test/CoWSwapEthFlow.t.sol

```diff
@@ -31,6 +33,35 @@ contract TestCoWSwapEthFlow is Test, ICoWSwapOnchainOrders {
         );
     }
 
+    // Unfortunately, even if the order mapping takes a bytes32 and returns a struct, Solidity interprets the output
+    // struct as a tuple instead. This wrapping function puts back the ouput into the same struct.
+    function ordersMapping(bytes32 orderHash)
```

Comment:
It's okay to collect all the helpers in this contract. But then maybe I would put the `testDeploymentParams` into a new contract called: `TestDeploymentSetup`